src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -13,7 +13,7 @@ case class RawlsUserSubjectId(value: String) extends UserAuthType
 case class RawlsGroupName(value: String) extends UserAuthType
 case class RawlsGroupEmail(value: String) extends UserAuthType
 case class RawlsBillingProjectName(value: String) extends UserAuthType
-case class RawlsGroupMemberList(userEmails: Seq[String], subGroupEmails: Seq[String])
+case class RawlsGroupMemberList(userEmails: Option[Seq[String]] = None, subGroupEmails: Option[Seq[String]] = None, userSubjectIds: Option[Seq[String]] = None, subGroupNames: Option[Seq[String]] = None)
```

Comment:
I agree with the use of Options, but I don't like the "if you have one of these then pass it under this field and if you have one of those then pass it under that field". We're capable of distinguishing between them, and IMO "be liberal in what you accept from others" applies here.

Since this is a philosophical question, should we run this by Silver (since they're our customers)?